cmd/minishift/cmd/start.go

```diff
@@ -127,6 +128,12 @@ func runStart(cmd *cobra.Command, args []string) {
 	setOcProxy()
 	setShellProxy()
 
+	if viper.IsSet("http-proxy") {
```

Comment:
My guess is that since they are only using one setting for the proxy, they assume it is a proxy which can do both. The distinction between HTTP and HTTPS proxies as we do, is a bit unfortunate imo, but we are basically aligning with _cluster up_ here. I would be very surprised if subscription-manager send passwords and so on plain text over the wire. What's the log showing? You should be seeing in the proxy logs which type of proxy requests are getting made, right? Our test setup falls short here as well. We should have two test proxies, one for HTTP and one for HTTPS.

Also, it is absolutely essential that we work out some integration tests around this, which are independent of some existing infrastructure. We are about to introduce the new integration test framework. Once it is there, there is no excuse to not this. 

> I have not found any documentation for subscription manager that it uses https.

Did you check the proxy logs?

>  I suggest we create an followup issue and findout more about it.

I guess.